components/org.wso2.carbon.metrics.view.ui/src/main/resources/web/metrics-view/hierarchy.jsp

```diff
@@ -0,0 +1,324 @@
+<%--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@prabathabey It was already been fixed in [commit:d8ad48](https://github.com/grainier/carbon-metrics/commit/d8ad48ed53fd2c90ebcfe5923a3ae3e00215bc43).